app/functions/cluster-config/loader/command.go

```diff
@@ -0,0 +1,237 @@
+	if err != nil {
+		log.Ctx(c.Context).Err(err).Msg("failed to create the validator config")
+		errs = append(errs, err.Error())
+	} else {
+		enc, err2 := settingsValidator.ToBytes()
+		if err2 != nil {
+			log.Ctx(c.Context).Err(err2).Msg("failed to encode the validator config to bytes")
+			errs = append(errs, fmt.Errorf("failed to encode the settings: %w", err2).Error())
+		} else {
+			settingsValidatorB64 = base64.StdEncoding.EncodeToString(enc)
+		}
+	}
+
+	// parse the validator config
+	settingsWebhookB64 := ""
+	settingsWebhook, err := cfg_webhook.NewSettings(c.StringSlice(FlagconfigWebhook)...)
+	if err != nil {
+		log.Ctx(c.Context).Err(err).Msg("failed to create the webhook config")
+		errs = append(errs, err.Error())
+	} else {
+		enc, err2 := settingsWebhook.ToBytes()
+		if err2 != nil {
+			log.Ctx(c.Context).Err(err2).Msg("failed to encode the webhook config to bytes")
+			errs = append(errs, fmt.Errorf("failed to encode the settings: %w", err2).Error())
+		} else {
+			settingsWebhookB64 = base64.StdEncoding.EncodeToString(enc)
+		}
+	}
+
+	// parse the validator config
+	settingsAggregatorB64 := ""
+	settingsAggregator, err := cfg_gator.NewSettings(c.StringSlice(FlagConfigAggregator)...)
+	if err != nil {
+		log.Ctx(c.Context).Err(err).Msg("failed to create the aggregator config")
+		errs = append(errs, err.Error())
+	} else {
+		enc, err2 := settingsAggregator.ToBytes()
+		if err2 != nil {
+			log.Ctx(c.Context).Err(err2).Msg("failed to encode the gator config to bytes")
+			errs = append(errs, fmt.Errorf("failed to encode the settings: %w", err2).Error())
+		} else {
+			settingsAggregatorB64 = base64.StdEncoding.EncodeToString(enc)
+		}
+	}
+
+	// create a new cluster config object
+	cfg := clusterconfig.ClusterConfig{
+		Account:                   c.String(FlagAccount),
+		Region:                    c.String(FlagRegion),
+		Namespace:                 ns,
+		ProviderId:                providerID,
+		ClusterName:               c.String(FlagClusterName),
+		K8SVersion:                k8sVersion,
+		DeploymentName:            c.String(FlagDeployName),
+		ChartVersion:              c.String(FlagChartVersion),
+		AgentVersion:              c.String(FlagAgentVersion),
+		ConfigValuesBase64:        c.String(FlagValuesB64),
+		ConfigValidatorBase64:     settingsValidatorB64,
+		ConfigWebhookServerBase64: settingsWebhookB64,
+		ConfigAggregatorBase64:    settingsAggregatorB64,
+		Errors:                    errs,
+	}
+
+	// print out the rendered cluster config
+	enc, err := json.MarshalIndent(&cfg, "", "  ")
+	if err == nil {
+		log.Ctx(c.Context).Debug().Msg("Rendered ClusterConfig:")
+		fmt.Println(string(enc))
+	}
+
+	// ship to the remote
+	if err := post(c.Context, settingsValidator, &cfg); err != nil {
+		log.Ctx(c.Context).Err(err).Msg("failed to ship the config to the remote")
+	}
+
+	// should not return an error in most cases
+	return nil
+}
+
+func post(
```

Comment:
does this partially duplicate the post [here](https://github.com/Cloudzero/cloudzero-agent/blob/develop/app/utils/telemetry/post.go#L37)? or, maybe it doesn't make sense to re-use given that we're adding a custom header?